setup.py

```diff
@@ -0,0 +1,75 @@
+from setuptools import setup, find_packages
+
+__version__ = "0.1.0"
+
+# More information on setting values:
+# https://github.com/Sceptre/project/wiki/sceptre-template-handler-template
+
+# lowercase, use `-` as separator.
+TEMPLATE_HANDLER_NAME = 'custom-template-handler'
+# the template_handler call in sceptre e.g. !command_name.
+TEMPLATE_HANDLER_COMMAND_NAME = 'custom_template_handler'
```

Comment:
This isn't how template handlers are used. Perhaps
```suggestion
# the template_handler call in sceptre e.g. !command_name.
TEMPLATE_TYPE = 'custom_template_handler'
```

Reply:
done.